### PR TITLE
docs: change README to show Write veneer

### PR DESCRIPTION
**Pull request by alvarowolfx**
This PR adds a workaround to show Storage Write API as example on the README of this repo by adding another sample on the `quickstart.js` file. 

We still keep the `bigquerystorage_quickstart` sample to use the Read API, as that's what is being used in all languages and docs.

### CODE REVIEW

File: .readme-partials.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      const tableId = 'my_table';
+
+      const destinationTable = `projects/${projectId}/datasets/${datasetId}/tables/${tableId}`;
+      const writeClient = new WriterClient({projectId});
+      const bigquery = new BigQuery({projectId: projectId});
+
+      try {
+        const dataset = bigquery.dataset(datasetId);
+        const table = await dataset.table(tableId);
+        const [metadata] = await table.getMetadata();
```
Comment (yirutang) [inline .readme-partials.yaml:36]: Let's go with GetWriteStream here.

File: .readme-partials.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          'root'
+        );
+
+        const connection = await writeClient.createStreamConnection({
+          streamId: managedwriter.DefaultStream,
+          destinationTable,
+        });
+        const streamId = connection.getStreamId();
+
+        const writer = new JSONWriter({
```
Comment (yirutang) [inline .readme-partials.yaml:51]: enable retry?

Comment (alvarowolfx) [reply .readme-partials.yaml:52]: added enable retry

File: .readme-partials.yaml (hunk 1 above)
Comment (alvarowolfx) [reply .readme-partials.yaml:37]: I'll add once #453 is merged

**Review by yirutang — APPROVED** [review]

File: .readme-partials.yaml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+    The BigQuery Storage Write API is a unified data-ingestion API for BigQuery. 
+    It combines streaming ingestion and batch loading into a single high-performance API.
+    You can use the Storage Write API to stream records into BigQuery in real time or 
+    to batch process an arbitrarily large number of records and commit them in a single 
+    atomic operation. 
+
+    Read more in our [introduction guide](https://cloud.google.com/bigquery/docs/write-api).
+
+    ```javascript
```
Comment (shollyman) [inline .readme-partials.yaml:16]: Maybe worth explaining what this sample does at a high level in the preface?

Something like "Using a system provided default stream, this code sample demonstrates using the schema of a destination stream/table to construct a writer, and send several batches of row data to the table."

File: .readme-partials.yaml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      } catch (err) {
+        console.log(err);
+      } finally {
+        writeClient.close();
+      }
+    }
+    ```
+
+    The BigQuery Storage Read API provides fast access to BigQuery-managed storage by 
+    using an rpc-based protocol. When you use the Storage Read API, structured data is
```
Comment (shollyman) [inline .readme-partials.yaml:103]: s/rpc/gRPC/

**Review by shollyman — APPROVED** [review]
mostly just comments feedback.  I didn't copy all comments to all files but they're likely relevant.

**Pull request merged**